renderdoc/driver/gl/gl_hooks_egl.cpp

```diff
@@ -435,12 +435,7 @@ __attribute__((visibility("default"))) EGLContext eglCreateContext(EGLDisplay di
   init.depthBits = value;
   eglhooks.eglGetConfigAttrib_real(display, config, EGL_STENCIL_SIZE, &value);
   init.stencilBits = value;
-// TODO: how to detect this?
-#if ENABLED(RDOC_ANDROID)
   init.isSRGB = 0;
```

Comment:
IIUC, the problem with checking for EGL_KHR_gl_colorspace is that it is only for default framebuffers. 

I just tested Mesa and it does actually support EGL_KHR_gl_colorspace, so I pushed the EGL_GL_COLORSPACE_KHR=EGL_GL_COLORSPACE_SRGB_KHR attrib pair. This worked fine so I set isSRGB to true, but when I replayed the capture of course it creates the m_FakeBB_FBO **non-default** framebuffer. This reported the error:
'GL_INVALID_FRAMEBUFFER_OPERATION in glClear(incomplete framebuffer)'

So whilst I can create sRGB default EGL framebuffers, the replay code still wont work because that extension still wont let us FBOs with sRGB textures.

So would you agree that leaving a comment to check for EGL_KHR_gl_colorspace would be a bit misleading?